replica/table.cc

```diff
@@ -577,7 +577,7 @@ table::update_cache(compaction_group& cg, lw_shared_ptr<memtable> m, std::vector
             add_sstable(cg, sst);
         }
         m->mark_flushed(std::move(new_ssts_ms));
-        try_trigger_compaction();
+        try_trigger_compaction(cg);
     });
     if (cache_enabled()) {
```

Comment:
Feels like this function should be moved to compaction_group.

Q: should we move row_cache to compaction_group too?

We don't need that to support tablets (we have invalidate_range) but maybe it makes things simpler.

Reply:
It reduces efficiency too, so maybe not.